.github/workflows/build_for_openwrt.yml

```
@@ -48,26 +48,26 @@ jobs:
     - name: Harden the runner (Audit all outbound calls)
-      uses: step-security/harden-runner@20cf305ff2072d973412fa9b1e3a4f227bda3c76 # v2.14.0
+      uses: step-security/harden-runner@e3f713f2d8f53843e71c69a996d56f51aa9adfb9 # v2.14.1
       with:
         egress-policy: audit
 
-    - uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # v6.0.1
+    - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
       with:
         fetch-depth: 0
     - name: Ensure git safe directory
       run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2046:warning:1:42: Quote this to prevent word splitting

(shellcheck)